site/source/docs/porting/asyncify.rst

```diff
@@ -210,6 +210,8 @@ the list of imports to the wasm module that the Asyncify instrumentation must be
 aware of. Giving it that list tells it that all other JS calls will **not** do
 an async operation, which lets it not add overhead where it isn't needed.
 
+.. note:: If the import is not inside ``env`` the full path must be specified, for example, ``ASYNCIFY_IMPORTS=["wasi_snapshot_preview1.fd_write"]``
```

Comment:
Can you drop the square brackets and the quotes. e.g. just `ASYNCIFY_IMPORTS=wasi_snapshot_preview1.fd_write`

Reply:
Yep, just updated it